BCStudentSoftwareDevTeam/celts
# Adds Tabs to Link Service Transcript and Upcoming Events

This PR fixes issue #167 where there was no link to the volunteer service transcript page. 

There is now a nav-tab in both serviceTranscript.html and volunteerProfile.html. I also added and reformatted their names to better reflect the wireframe. 

TO TEST:
Go to /profile/<USERNAME> 
There should be a nav-tab that takes the user to the username's transcript. 
Click on "Service Transcript" 
It sends the user to /<USERNAME>/serviceTranscript
